src/Drupal/Commands/core/LocaleCommands.php

```diff
@@ -128,4 +131,83 @@ public function update($options = ['langcodes' => self::REQ])
 
         drush_backend_batch_process();
     }
+
+    /**
+     * Exports to a gettext translation file.
+     *
+     * @command locale:export
+     *
+     * @param string $langcode The language code of the exported translations.
+     * @option $include
+     * @usage drush locale:export nl > nl.po
+     *   Export the Dutch non-customized translations.
+     * @usage drush locale:export nl --include=customized > nl.po
+     *   Export the Dutch customized translations.
+     * @usage drush locale:export > drupal.pot
+     *   Export the template file to translate.
+     * @see \Drupal\locale\Form\ExportForm::submitForm
+     *
+     * @throws \Exception
+     * @return string
+     *
+     * @aliases locale-export
+     * @drupal-dependencies locale
+     */
+    public function export($langcode = null, $options = ['include' => ['customized']])
+    {
+        // If template is required, language code is not given.
+        if ($langcode != LanguageInterface::LANGCODE_SYSTEM) {
+            $language = \Drupal::languageManager()->getLanguage($langcode);
+        } else {
+            $language = null;
+        }
+
+        $includes_allowed = [
+            'non_customized',
+            'customized',
+            'not_translated',
+        ];
+
+        foreach ($options['include'] as $include) {
+            if (!in_array($include, $includes_allowed)) {
+                throw new \Exception(dt('Include must contain one of: @types.', ['@types' => implode(', ', $includes_allowed)]));
+            }
+        }
+
+        $content_options = is_array($options['include']) ? $options['include'] : [$options['include']];
```

Comment:
You don't trust here that $options['include'] is an array, but at line 171 you do rely on it being an array. Do it once, and early.